data/locale/es-ES.ini

```diff
@@ -0,0 +1,22 @@
+AudioMonitor="Monitor de audio"
+Description="Panel y filtro de monitorización de audio"
+Device="Dispositivo"
+Volume="Volumen"
+Default="Por defecto"
+Locked="Bloqueado"
+Linked="Vinculado al volumen de la fuente"
+VolumeOutput="Volumen de salida"
+MeterOutput="Ver nivel de la salida"
+OnlyActive="Sólo activas"
+OutputSlider="Mostrar todas las salidas"
+Outputs="Pistas de audio"
+Track="Pista"
+Show="Mostrar"
+Format="Formato"
+UInt8="8 bits"
+Int16="16 bits"
+Int32="32 bits"
+Float32="32 bits punto flotante"
+SampleRate="Frecuencia de muestreo"
+Delay=Retardo
+Port=Puerto
```

Comment:
Delay and port don't have quotes

Reply:
I forgot the quotes, you are right.

Updating...